PGM/src/main/java/tc/oc/pgm/mutation/types/uhc/BarebonesScenario.java

```diff
@@ -0,0 +1,96 @@
+package tc.oc.pgm.mutation.types.uhc;
+
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.inventory.CraftItemEvent;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.inventory.CraftingInventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.event.entity.PlayerDeathEvent;
+import tc.oc.pgm.match.Match;
+import tc.oc.pgm.mutation.Mutation;
+import tc.oc.pgm.mutation.types.UHCMutation;
+
+public class EnchantlessScenario extends UHCMutation.Impl {
+    public EnchantlessScenario(Match match, Mutation mutation) {
+        super(match, mutation);
+    }
+
+    // Disable drops for ores that are tiered above iron
+    @EventHandler(priority = EventPriority.HIGH)
+    public void onBlockBreak(BlockBreakEvent event) {
+        Block block = event.getBlock();
+
+        if (block.getType() == Material.REDSTONE_ORE) {
```

Comment:
This should be combined into one large logical block, since all of the below statements. 

This does the same thing as the block drop modification module which isn't ideal, but that item doesn't support messages. Later we should support that instead of this, but this will do for now.